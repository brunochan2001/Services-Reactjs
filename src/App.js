import React from 'react';
import { Provider } from 'react-redux';
import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
import {
  Error404,
  ServiceAuto,
  ServiceHealth,
  ServiceHome,
  ServicesPage
} from './pages';
import store from './store';

const App = () => {
  return (
    <>
      <Provider store={store}>
        <Router>
          <Routes>
            <Route exact path="/" element={<ServicesPage />} />
            <Route exact path="/Auto" element={<ServiceAuto />} />
            <Route exact path="/Salud" element={<ServiceHealth />} />
            <Route exact path="/Hogar" element={<ServiceHome />} />
            <Route exact path="*" element={<Error404 />} />
          </Routes>
        </Router>
      </Provider>
    </>
  );
};

export default App;
