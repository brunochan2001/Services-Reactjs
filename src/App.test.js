import { render } from '@testing-library/react';
import App from './App';
test('debe de hacer match con el snapshot', () => {
  const { container } = render(<App />);
  expect(container).toMatchSnapshot();
});
