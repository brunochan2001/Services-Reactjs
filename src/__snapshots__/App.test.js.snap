// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`debe de hacer match con el snapshot 1`] = `
<div>
  <header>
    <header
      class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionStatic css-hip9hq-MuiPaper-root-MuiAppBar-root"
    >
      <div
        class="MuiContainer-root MuiContainer-maxWidthXl css-19r6kue-MuiContainer-root"
      >
        <div
          class="MuiToolbar-root MuiToolbar-regular css-r6ewbb-MuiToolbar-root"
        >
          <div
            class="MuiBox-root css-1lvtzne"
          >
            <button
              aria-controls="menu-appbar"
              aria-haspopup="true"
              aria-label="account of current user"
              class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit MuiIconButton-sizeLarge css-1pz7awu-MuiButtonBase-root-MuiIconButton-root"
              tabindex="0"
              type="button"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                data-testid="MenuIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
                />
              </svg>
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
          </div>
          <div
            class="MuiBox-root css-1t6c9ts"
          >
            <a
              class="appbar-link"
              href="/"
            >
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium css-1q39md6-MuiButtonBase-root-MuiButton-root"
                tabindex="0"
                type="button"
              >
                Inicio
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
            </a>
            <a
              class="appbar-link"
              href="/Salud"
            >
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium css-1q39md6-MuiButtonBase-root-MuiButton-root"
                tabindex="0"
                type="button"
              >
                Salud
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
            </a>
            <a
              class="appbar-link"
              href="/Hogar"
            >
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium css-1q39md6-MuiButtonBase-root-MuiButton-root"
                tabindex="0"
                type="button"
              >
                Hogar
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
            </a>
            <a
              class="appbar-link"
              href="/Auto"
            >
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium css-1q39md6-MuiButtonBase-root-MuiButton-root"
                tabindex="0"
                type="button"
              >
                Auto
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
            </a>
          </div>
        </div>
      </div>
    </header>
    <div />
  </header>
  <main
    class="main"
  >
    <div
      class="MuiTypography-root MuiTypography-h4 MuiTypography-gutterBottom title css-1vw6mcs-MuiTypography-root"
    >
      Servicios
    </div>
    <div
      class="spin-container"
    >
      <span
        class="MuiCircularProgress-root MuiCircularProgress-indeterminate MuiCircularProgress-colorPrimary css-18lrjg1-MuiCircularProgress-root"
        role="progressbar"
        style="width: 40px; height: 40px;"
      >
        <svg
          class="MuiCircularProgress-svg css-1idz92c-MuiCircularProgress-svg"
          viewBox="22 22 44 44"
        >
          <circle
            class="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate css-176wh8e-MuiCircularProgress-circle"
            cx="44"
            cy="44"
            fill="none"
            r="20.2"
            stroke-width="3.6"
          />
        </svg>
      </span>
    </div>
  </main>
  <div />
</div>
`;

exports[`renders learn react link 1`] = `
<div>
  <header>
    <header
      class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionStatic css-hip9hq-MuiPaper-root-MuiAppBar-root"
    >
      <div
        class="MuiContainer-root MuiContainer-maxWidthXl css-19r6kue-MuiContainer-root"
      >
        <div
          class="MuiToolbar-root MuiToolbar-regular css-r6ewbb-MuiToolbar-root"
        >
          <div
            class="MuiBox-root css-1lvtzne"
          >
            <button
              aria-controls="menu-appbar"
              aria-haspopup="true"
              aria-label="account of current user"
              class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit MuiIconButton-sizeLarge css-1pz7awu-MuiButtonBase-root-MuiIconButton-root"
              tabindex="0"
              type="button"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                data-testid="MenuIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
                />
              </svg>
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
          </div>
          <div
            class="MuiBox-root css-1t6c9ts"
          >
            <a
              class="appbar-link"
              href="/"
            >
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium css-1q39md6-MuiButtonBase-root-MuiButton-root"
                tabindex="0"
                type="button"
              >
                Inicio
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
            </a>
            <a
              class="appbar-link"
              href="/Salud"
            >
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium css-1q39md6-MuiButtonBase-root-MuiButton-root"
                tabindex="0"
                type="button"
              >
                Salud
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
            </a>
            <a
              class="appbar-link"
              href="/Hogar"
            >
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium css-1q39md6-MuiButtonBase-root-MuiButton-root"
                tabindex="0"
                type="button"
              >
                Hogar
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
            </a>
            <a
              class="appbar-link"
              href="/Auto"
            >
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium css-1q39md6-MuiButtonBase-root-MuiButton-root"
                tabindex="0"
                type="button"
              >
                Auto
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
            </a>
          </div>
        </div>
      </div>
    </header>
    <div />
  </header>
  <main
    class="main"
  >
    <div
      class="MuiTypography-root MuiTypography-h4 MuiTypography-gutterBottom title css-1vw6mcs-MuiTypography-root"
    >
      Servicios
    </div>
    <div
      class="spin-container"
    >
      <span
        class="MuiCircularProgress-root MuiCircularProgress-indeterminate MuiCircularProgress-colorPrimary css-18lrjg1-MuiCircularProgress-root"
        role="progressbar"
        style="width: 40px; height: 40px;"
      >
        <svg
          class="MuiCircularProgress-svg css-1idz92c-MuiCircularProgress-svg"
          viewBox="22 22 44 44"
        >
          <circle
            class="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate css-176wh8e-MuiCircularProgress-circle"
            cx="44"
            cy="44"
            fill="none"
            r="20.2"
            stroke-width="3.6"
          />
        </svg>
      </span>
    </div>
  </main>
  <div />
</div>
`;
